### DISCUSSION TITLE
User does not see me and my assets

### DISCUSSION THREAD

**Discussion by TausendMedien** [mozilla/hubs#2716] [General Questions & Discussion] [answered]
Hi, 
I tested mozilla hubs several times and love it. But twice at least one of the participants was not seeing all others. He was also not seeing the objects they created. In both instances, this was rather problematic, as I had created the rooms and some assets (Youtube Videos, sharing my screen) and they could not see me and my assets. I also posted this question in Discord before I noticed it might fit better here.

Keep up the good work!
Uli

**Comment by emclaren** [comment, marked as answer]
Hi Uli, thanks for posting it here! as you probably saw, we are excited to be moving our support off Discord so it's easier to see when questions have been answered, and make them easier to search. 

This is definitely not the user experience we are going for! Are you able to tell me a little bit more to help us troubleshoot/report this bug (if possible)?

- Do you know the type of device they were using (mobile device/Windows computer/ Mac computer/ VR Headset)? It would be helpful for troubleshooting to know their operating system version as well, but I totally understand if you don't have access to this. 

- Do you know the web browser they were using (firefox/chrome/edge/safari/etc)?

- Were they able to see anyone else (or the media created by other users) in the room? 

- Was it the same participant this happened with, or were there two unique participants that had this happen?

- Did they try refreshing the URL to see if it still happened when they re-entered the room? 


Thank you for reporting this.

**Reply by TausendMedien** [reply]
Thank you for your reply. It will be hard to get info on this. I will monitor my next session more closely and report better future errors.

**Reply by emclaren** [reply]
Hey @TausendMedien - thanks, I totally understand. 

We are definitely having something weird occasionally coming up with WebRTC that could totally be related to what you are describing. It's a bit hard to track because it happens so randomly. We have a form that would be a huge help if you could complete if you spot any issues like this (it will help us spot any possible patterns): https://docs.google.com/forms/d/e/1FAIpQLSd_dw8uQN2VjxLcxXmXFzzaJVU8IWXuUAnRJd1kjoB_TohqsA/viewform
